src/meta/src/stream/source_manager.rs

```diff
@@ -775,7 +779,7 @@ impl SourceManager {
         let current_splits_ref = splits.clone();
         let source_id = source.id;
 
-        let connector_properties = extract_prop_from_source(source)?;
+        let connector_properties = extract_prop_from_source(source, true)?;
```

Comment:
highlight `true` here

Reply:
How about using two functions to clearly express the difference in behavior? For example, one named `extract_prop_from_existing_source` and the other `extract_prop_from_new_source`.

Reply:
I think better naming to `extract_prop_allow_unknown_args` and `extract_prop_deny_unknown_args`, although sounds like java

Reply:
Why deny unknown fields here and not in `create_source_worker_async`? May add some comments to describe the difference.

Reply:
Comments are on `extract_prop_from_source`